resources/styles/global/forms.less

```diff
@@ -1,3 +1,5 @@
+@form-error-color: #E35C3B;
```

Comment:
I think the goal was to *at least* move the colors up to the top of the file (to ease refactoring) but the eventual goal is to have all the colors in one place. Since this is a new file I think it should be up in variables.